spec/models/snap_application_spec.rb

```diff
@@ -183,4 +183,33 @@
       end
     end
   end
+
+  describe "#drive_status" do
+    context "when there are drive attempts and no errors" do
+      it "returns :drive_none" do
+        snap_application = create(:snap_application)
+
+        expect(snap_application.drive_status).to eq :drive_none
+      end
+    end
+
+    context "when there are drive attempts and no errors" do
+      it "returns :drive_in_flight" do
+        driven_application = create(:driver_application)
+        snap_application = driven_application.snap_application
+
+        expect(snap_application.drive_status).to eq :drive_in_flight
+      end
+    end
+
+    context "when there are drive attempts and no errors" do
```

Comment:
I think this spec description needs an update 🐦 👁

Reply:
aye aye aye